assets/css/modules/_landing.scss

```diff
@@ -61,6 +61,11 @@
   .play-store-badge {
     height: 60px;
   }
+  @media (max-width: 992px) {
```

Comment:
https://github.com/blockchain/My-Wallet-V3-Frontend/blob/master/assets/css/modules/_home.scss#L12

I looked at this. my mistake